Text/XHtml/Internals.hs

```diff
@@ -109,17 +134,24 @@ instance (CHANGEATTRS b) => CHANGEATTRS (a -> b) where
       changeAttrs fn f = \ arg -> changeAttrs (fn arg) f
 
 instance ADDATTRS Html where
-      (Html htmls) ! attr = Html (map addAttrs htmls)
-        where
-              addAttrs (html@(HtmlTag { markupAttrs = attrs }) )
-                            = html { markupAttrs = attrs ++ attr }
```

Comment:
So here we have `attrs ++ attr`, with `attr` being the new thing. So:

```haskell
(anchor ! [href "asdf"]) ! [theclass "woop"]
```

should evaluate to:

```haskell
(anchor ! [href "asdf"]) ! [theclass "asdf"]
(Html { attrs = [href "asdf"] }) ! [theclass "asdf"]
(Html { attrs = [href "asdf"] ++ [theclass "asdf"] })
```
```